pandas/core/ops/__init__.py

```diff
@@ -57,6 +57,7 @@
     _make_flex_doc,
     _op_descriptions,
 )
```

Comment:
use absolute imports

Reply:
I think we're holding off on these until the `import pandas as pd` is removed from this file

Reply:
that's orthogonal. these should not have been like this in the first place.

Reply:
updated

Reply:
thxs, ping on green.